src/main/java/com/akademia/projectplanner/mapper/UserMapper.java

```diff
@@ -24,7 +24,7 @@ public UserEntity toUserEntity(UserDto userDto) {
     userEntity.setName(userDto.getName());
     userEntity.setEmail(userDto.getEmail());
     userEntity.setPassword(new BCryptPasswordEncoder().encode(userDto.getPassword()));
-    userEntity.setRole(Role.DEVELOPER);
+    userEntity.setRole(Role.ADMINISTRATOR);
```

Comment:
dlaczego zmieniamy na automatycznego administratora dla wszystkich? :) ogólnie to było rozwiązanie tymczasowe z ustawianiem roli w mapperze, dodam nowe zadanie na to